data/pokemon_upgrade_cost.json

```diff
@@ -0,0 +1,82 @@
+[
+[1, 200],
+[1, 200],
```

Comment:
As far as I know, that's true. So we're using the entry index as our level and the first index in the entry as our level "group"?